src/test/scala/fairysf/breakthroughtroyka/MiniBreakthroughTroykaFenTest.scala

```diff
@@ -0,0 +1,16 @@
+package strategygames.fairysf
```

Comment:
would extend this file to cover FEN tests for both Breakthrough and Mini Breakthrough